packages/backend/package.json

```diff
@@ -2,7 +2,7 @@
     "name": "cloudify-ui-common-backend",
     "version": "0.0.1",
     "description": "Common Cloudify UI backend library",
-    "homepage": "https://github.com/cloudify-cosmo/cloudify-ui-common/blob/master/backend/README.md",
+    "homepage": "https://github.com/cloudify-cosmo/cloudify-ui-common/blob/master/packages/backend/README.md",
```

Comment:
Not related, but as I spotted that after publishing prerelease version of `cloudify-ui-common-scripts` in NPM (https://www.npmjs.com/package/cloudify-ui-common-scripts) I decided to fix all packages at once.